karma.conf.js

```diff
@@ -7,6 +7,7 @@ module.exports = function(config) {
       'node_modules/angular/angular.js',
       'node_modules/angular-mocks/angular-mocks.js',
       'js/**/*.js',
+      'spec/**/*.js',
```

Comment:
@etgrieco having *.js and *.spec.js is redundant. i had *.spec.js so that we were explicitly labeling our tests but you can do either.

Reply:
Yep, makes sense